### PR TITLE
Regeneration of services

### PR DESCRIPTION
**Pull request by anweshan**

### CODE REVIEW

**Comment by anweshan** [comment]
Travis will not pass until watson-developer-cloud is released FYI, so review should be held off until then

**Comment by bvennam** [comment]
@anweshan take a peek at the test file in `/tests/src/test/scala/packages/AssistantV1Tests.scala` 

It's building out strings for the expected action names -- and also uses these names to delete the actions out of the package before deleting the package.  Since you're adding a new action `list-mentions` you'll also need to delete that action from the package before the package can be deleted.  

It would make sense to consider whether these test files can also be generated, rather than written manually.

File: packages/assistant-v1/README.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ The Watson Assistant V1 Package will contain the following entities. Find additi
...
 | [list-entities](https://www.ibm.com/watson/developercloud/assistant/api/v1/curl.html?curl#list-entities) | action |  username, password,  iam_access_token, iam_apikey, iam_url,  headers, headers[X-Watson-Learning-Opt-Out], url,    workspace_id,     export,     page_limit,     include_count,     sort,     cursor,     include_audit,  | List entities |
 | [update-entity](https://www.ibm.com/watson/developercloud/assistant/api/v1/curl.html?curl#update-entity) | action |  username, password,  iam_access_token, iam_apikey, iam_url,  headers, headers[X-Watson-Learning-Opt-Out], url,    workspace_id,     entity,    new_entity, new_description, new_metadata, new_fuzzy_match, new_values,  | Update entity |
+| [list-mentions](https://www.ibm.com/watson/developercloud/assistant/api/v1/curl.html?curl#list-mentions) | action |  username, password,  iam_access_token, iam_apikey, iam_url,  headers, headers[X-Watson-Learning-Opt-Out], url,    workspace_id,     entity,     export,     include_audit,  | List entity mentions |
```
Comment (bvennam) [inline packages/assistant-v1/README.md:36]: list-mentions will need to be added to the scala tests for AssistantV1.

File: packages/visual-recognition-v3/actions/list-classifiers.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ function main(params) {
     const _params = getParams(
       params,
       'watson-vision-combined',
-      'watson_vision_combined'
+      'watson_vision_combined',
```
Comment (mediumTaj) [inline packages/visual-recognition-v3/actions/list-classifiers.js:41]: Is this change intentional?

**Review by mediumTaj — APPROVED** [review]
Just one question about a stray comma
